lib/matplotlib/patches.py

```diff
@@ -1840,7 +1840,7 @@ def set_width(self, width):
         ----------
         width : float
         """
-        if min(self.a, self.b) <= width:
+        if width <= min(self.a, self.b) is False:
             raise ValueError(
                 'Width of annulus must be less than or equal semi-minor axis')
```

Comment:
While you're here, could you also fix the typo? There should be a 'to' between equal and semi-minor.
```suggestion
                'Width of annulus must be less than or equal to semi-minor axis')
```